src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -584,6 +584,7 @@ public List<RESTZaakBetrokkene> listBetrokkenenVoorZaak(@PathParam("uuid") final
     @Path("communicatiekanalen")
     public List<RESTCommunicatiekanaal> listCommunicatiekanalen() {
         final List<CommunicatieKanaal> communicatieKanalen = vrlClientService.listCommunicatiekanalen();
+        communicatieKanalen.removeIf(communicatieKanaal -> communicatieKanaal.getNaam().equals("E-formulier"));
```

Comment:
Idem

Reply:
Opgelost in https://github.com/NL-AMS-LOCGOV/zaakafhandelcomponent/pull/1467